src/map/clif.h

```diff
@@ -724,7 +725,9 @@ struct clif_interface {
 	void (*mvp_noitem) (struct map_session_data* sd);
 	void (*changed_dir) (struct block_list *bl, enum send_target target);
 	void (*charnameack) (int fd, struct block_list *bl);
+#ifdef ENABLE_MONSTER_HP_BAR
```

Comment:
Same as above, ifdef shouldn't be here.
Function content should be enclosed in ifdef